### PR TITLE
MLDB-1802 go to readthrough in orderby context if nested column is no…

### PR DESCRIPTION
**Pull request by steadtler**
…t found.

Turn out the issue had nothing to do with joins after all.

### CODE REVIEW

File: testing/MLDB-1802-select-orderby.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# MLDB-1802-join-order-by.py
+# Mathieu Marquis Bolduc, 2016-12-07
```
Comment (FinchPowers) [inline testing/MLDB-1802-select-orderby.py]: Mauvais format de date.

Comment (FinchPowers) [inline testing/MLDB-1802-select-orderby.py]: Où tu es dans le futur? :p

File: testing/MLDB-1802-select-orderby.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    ["row_b", 1],
+                    ["row_a", 1]]
+
+        expected2 = [["_rowName","dataset1.y"],
+                    ["row_c", 3],
+                    ["row_b", 2],
+                    ["row_a", 1]]
+
+        self.assertEqual(res1, expected1)
+        self.assertEqual(res2, expected2)
```
Comment (FinchPowers) [inline testing/MLDB-1802-select-orderby.py]: `self.assertTableResultEquals`

**Comment by FinchPowers** [comment]
Corrections mineures. +1 ensuite.

File: testing/MLDB-1802-select-orderby.py (hunk 1 above)
Comment (steadtler) [inline testing/MLDB-1802-select-orderby.py]: whynotboth.gif

**Pull request merged**